probe-rs-target/src/flash_algorithm.rs

```diff
@@ -16,10 +16,12 @@ pub struct RawFlashAlgorithm {
     pub description: String,
     /// Whether this flash algorithm is the default one or not.
     pub default: bool,
-    /// List of 32-bit words containing the position-independent code for the algo.
+    /// List of 32-bit words containing the code for the algo.
```

Comment:
I think we should still mention here that the code has to be position-independent *if* `load_address` is not set.